cura/CuraApplication.py

```diff
@@ -975,6 +994,8 @@ def registerObjects(self, engine):
         qmlRegisterSingletonType(SimpleModeSettingsManager, "Cura", 1, 0, "SimpleModeSettingsManager", self.getSimpleModeSettingsManager)
         qmlRegisterSingletonType(MachineActionManager.MachineActionManager, "Cura", 1, 0, "MachineActionManager", self.getMachineActionManager)
 
+        qmlRegisterType(WelcomePagesModel, "Cura", 1, 0, "WelcomePageModel")
```

Comment:
Please rename this to `WelcomePages` to be consistent with the rest. It's especially confusing to have the `s` omitted in this one. We don't usually call stuff `Model` when exposing them to QML because everything is a model.

Reply:
Fixed typo in a97410656